### PR TITLE
HAWKULAR-609 - Implemented the 'Export JDR' operation.

### PR DESCRIPTION
**Pull request by jpkrohling**

### CODE REVIEW

**Comment by jpkrohling** [comment]
Once this is reviewed/merged, I'll send the related PR for Agent and then to Hawkular. So, if Hawkular requires a version instead of a git-sha, then I'd also request a release of this module.

File: hawkular-command-gateway-war/src/main/java/org/hawkular/cmdgw/ws/command/feed/ExportJDRResponseCommand.java [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+            throws Exception {
+
+        String status = response.getStatus();
+        String msg = response.getMessage();
+        MsgLogger.LOG.infof("Export JDR completed. Status=[%s], Message=[%s]", status, msg);
+
+        // determine what UI client needs to be sent the message
+        String uiClientId;
+
+        // TODO: how do we get this?
```
Comment (pilhuhn) [inline hawkular-command-gateway-war/src/main/java/org/hawkular/cmdgw/ws/command/feed/ExportJDRResponseCommand.java]: Isn't the resolution of this TODO the code below?

Comment (jpkrohling) [inline hawkular-command-gateway-war/src/main/java/org/hawkular/cmdgw/ws/command/feed/ExportJDRResponseCommand.java]: No, the appropriate solution is yet to be defined. A discussion about it is happening on the -dev and on HAWKULAR-498 . In fact, this code is mostly what there's already present on the existing ExecuteOperation.

**Comment by ppalaga** [comment]
LGTM

**Pull request merged**